### PR TITLE
WINC-1315: Document steps for EUS-to-EUS upgrade

### PR DESCRIPTION
**Pull request by jrvaldes**
This PR proposes the steps to support  EUS-to-EUS upgrade with Windows nodes.

### CODE REVIEW

**Comment by jrvaldes** [comment]
/hold

Do not merge. Depends on the e2e test coverage for EUS-to-EUS upgrade

**Comment by jrvaldes** [comment]
/hold cancel

e2e test PR
- https://github.com/openshift/release/pull/56409

**Comment by saifshaikh48** [comment]
/lgtm

nice, can basically be copy/pasted to include into openshift docs

File: README.md [hunk 1]
```diff
@@ -213,6 +213,35 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Windows nodes Openshift EUS-to-EUS upgrade
+
+WMCO does support upgrading from one EUS version to another EUS version of OCP, the Windows nodes will be upgraded
```
Comment (sebsoto) [inline README.md:218]: It might be helpful to add a link for EUS, not required though.
 https://access.redhat.com/support/policy/updates/openshift-eus

**Review by sebsoto — APPROVED** [review]
LGTM besides minor comments

File: README.md [hunk 2]
```diff
@@ -213,6 +213,35 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Windows nodes Openshift EUS-to-EUS upgrade
+
+WMCO does support upgrading from one EUS version to another EUS version of OCP, the Windows nodes will be upgraded 
+from the staring EUS to the new EUS version while keeping the Windows workload in a healthy state with no disruptions.
```
[the remark this answers is not in the document]
Comment (jrvaldes) [reply README.md]: addressed https://github.com/openshift/windows-machine-config-operator/commit/a2f163ee7e0b14fe2ffa2bab605f3802782b0b98

File: README.md (hunk 1 above)
Comment (jrvaldes) [reply README.md]: addressed https://github.com/openshift/windows-machine-config-operator/commit/a2f163ee7e0b14fe2ffa2bab605f3802782b0b98

**Comment by jrvaldes** [comment]
> /hold
> 
> holding to allow comments to be addressed

/hold cancel

comments addressed

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/wicd-unit-vsphere

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator  ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/wicd-unit-vsphere

**Pull request merged**